composer.json

```diff
@@ -69,5 +69,7 @@
         "psr-4": {
             "GuzzleHttp\\Tests\\": "tests/"
         }
-    }
+    },
+    "prefer-stable": true,
+    "minimum-stability": "dev"
```

Comment:
Why is that needed?

Reply:
Better to just have the dev on the single dependency we want, otherwise now we run the risk of accidentally needing other dev deps and the CI not catching it.

Reply:
This allows us to do `vendor/package:^3.0` before 3.0 is released. 

That will come in handy when Guzzle officially supports `guzzlehttp/psr7:2.0` which should happen before 2.0 is released. 

It also has the benefit that we dont need to update the CI when 2.0 is released.

Reply:
Doing `@dev` on the end of the individual version constraint we want also allows us to do that.